tests/h/activity/query_test.py

```diff
@@ -175,20 +175,16 @@ def test_preserves_other_query_terms_for_user_search(
 
         unparse.assert_called_once_with({"tag": "foo"})
 
-    def test_does_nothing_with_non_matching_queries(self, pyramid_request, unparse):
+    def test_doesnt_raise_with_non_matching_queries(self, pyramid_request, unparse):
         query = MultiDict({"tag": "foo"})
 
-        result = check_url(pyramid_request, query, unparse=unparse)
-
-        assert result is None
+        check_url(pyramid_request, query, unparse=unparse)
 
-    def test_does_nothing_if_not_on_search_page(self, pyramid_request, unparse):
+    def test_doesnt_raise_if_not_on_search_page(self, pyramid_request, unparse):
         pyramid_request.matched_route.name = "group_read"
         query = MultiDict({"group": "abcd1234"})
 
-        result = check_url(pyramid_request, query, unparse=unparse)
-
-        assert result is None
+        check_url(pyramid_request, query, unparse=unparse)
```

Comment:
```suggestion
        result = check_url(pyramid_request, query, unparse=unparse)
        
        assert not result
```

Reply:
Same as before, pylint complained about ` assignment-from-none`, kept the assert now without an assignment